client/src/main/java/com/networknt/client/ssl/SSLUtils.java

```diff
@@ -0,0 +1,37 @@
+package com.networknt.client.ssl;
+
+import java.security.cert.CertificateException;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.networknt.utility.StringUtils;
+
+public class SSLUtils {
+	private static final Logger logger = LoggerFactory.getLogger(SSLUtils.class);
+	
+	public static Set<String> resolveTrustedNames(String trustedNames){
+		Set<String> nameSet = Arrays.stream(StringUtils.trimToEmpty(trustedNames).split(","))
+				.filter(StringUtils::isNotBlank)
+				.collect(Collectors.toSet());
```

Comment:
Not sure this is the best approach for providing trusted names as it's on the application level rather then the connection level. Ie. for my service i have to set the list of all trusted names as part of all connections that i will be making, rather then setting the individual trusted names for a specific connection i'm creating. Thoughts?

Reply:
Do you mean you want make the trustedNames a constant once loaded?

Reply:
No I mean that if an api is initiating requests multiple services they will have to list the trusted names in the application config, instead as specifically for the individual connections being made. Does that make sense?